ydb/core/grpc_services/ya.make

```diff
@@ -128,6 +127,7 @@ PEERDIR(
     ydb/library/mkql_proto
     ydb/library/persqueue/topic_parser
     ydb/library/yql/parser/pg_wrapper/interface
+    ydb/library/yql/providers/generic/connector/api/service
```

Comment:
Do we really need it after removing https://github.com/ydb-platform/ydb/pull/2976/files#diff-8979cfaa4575f06f00707f9519db651cee3c8f590c25691f5feffab110958019L8
?